sdk/appcenter-storage/src/main/java/com/microsoft/appcenter/storage/Storage.java

```diff
@@ -68,6 +68,8 @@
 
     private HttpClient mHttpClient;
 
+    private Context mConntext;
```

Comment:
do not know why other property name start with `m`... thus add m here. I thought `m` is short for mock?

Reply:
m is android convention for member (variable) as opposed to s for static.

Reply:
We use that convention everywhere except for JSON models.